ci/linux/bootstrap-sudo

```diff
@@ -1,14 +1,36 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-[ ${EUID} -eq 0 ] || {
-    sudo ${BASH_SOURCE[0]}
-    exit 0
-}
-
 SUPPORT_FIRECLOUD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
 source ${SUPPORT_FIRECLOUD_DIR}/sh/common.inc.sh
 
+DPKGS="$(cat <<-EOF
+bash
+bsdmainutils
+build-essential
+curl
+file
+git
+python-minimal
+rsync
+uuid-runtime
+EOF
+)"
+
+if [ "${SF_USE_SUDO:-}" = "false" ]
+then
+    echo Not using sudo, so please make sure these packages are installed by executing:
+    echo sudo apt install $DPKGS
+    exit 0
+fi
```

Comment:
Your choice to shortcircuit means that you consider everything else unnecessary (and it might be that some things are related to CI environments in general, or to Travis in specific), but it might not be so with the existing commands or future commands that will get added.
So the current PR doesn't just handle the case of "no sudo", but also introduces new execution branches.

There are many alternatives, but some more thought would be needed to pick one.

Reply:
PS: also, if you just `exit 0`, it means that the instructions are lost in a pile of stdout text